### PR TITLE
feat: new DynamicIllustration component

### PR DESCRIPTION
**Pull request by lisalupi**
## Type

- Feature
- Enhancement

#### What is expected?

New illustrations and new component in ultraviolet/illustrations
#### The following changes were made:

(Describe what you did)

1. New component `DynamicIllustration`: adapt the illustration to the theme

2. 11 new illustrations (+ dark variant) added

### CODE REVIEW

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by PetmePant — APPROVED** [review]

**Review by BABAK0T0 — APPROVED** [review]

File: packages/illustrations/src/components/DynamicIllustration/Illustrations.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import searchDark from '../../various/empty/search-dark.webp'
+import searchLight from '../../various/empty/search-light.webp'
+import dangerDark from '../../various/feedback/danger-dark.webp'
+import dangerLight from '../../various/feedback/danger-light.webp'
+import successDark from '../../various/feedback/success-dark.webp'
+import successLight from '../../various/feedback/success-light.webp'
+import warningDark from '../../various/feedback/warning-dark.webp'
+import warningLight from '../../various/feedback/warning-light.webp'
+
+export type IllustrationsKeys = {
```
Comment (matthprost) [inline packages/illustrations/src/components/DynamicIllustration/Illustrations.tsx:24]: Could you add a little comment here to explain why we do that so we won't forget over time 😄

File: packages/illustrations/src/components/DynamicIllustration/__stories__/List.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import type { StoryFn } from '@storybook/react'
+import { Stack, Text } from '@ultraviolet/ui'
+import { DynamicIllustration } from '..'
+import { ILLUSTRATIONS } from '../Illustrations'
+
+export const List: StoryFn = props => (
+  <Stack gap={1}>
+    {Object.keys(ILLUSTRATIONS.light).map(iconName => (
+      <Stack direction="row" gap={1} alignItems="center">
```
Comment (matthprost) [inline packages/illustrations/src/components/DynamicIllustration/__stories__/List.stories.tsx:9]: When doing a map in react you need to provide a key

File: packages/illustrations/src/components/DynamicIllustration/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  'data-testid'?: string
+  className?: string
+}
+/**
+ * DynamicIllustration is a component made to automate the render of illustrations to adapt them to the current theme (light/dark/darker).
+ */
+export const DynamicIllustration = ({
+  name,
+  width = 'auto',
+  height = 'auto',
```
Comment (matthprost) [inline packages/illustrations/src/components/DynamicIllustration/index.tsx:27]: Any reason to set it auto by default? What if we leave it undefined by default?

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by matthprost — APPROVED** [review]
Let's goo 🚀

**Pull request merged**